maven-scm-api/src/main/java/org/apache/maven/scm/ChangeSet.java

```diff
@@ -414,6 +419,53 @@ public String getTimeFormatted()
         return TIME_FORMAT.format( getDate() );
     }
 
+    /**
+     * Getter for property tags.
+     *
+     * @return Value of property author.
+     */
+    public List<String> getTags()
+    {
+        if ( tags == null )
+        {
+            return Collections.emptyList();
```

Comment:
I don't know how much this matters, but this returns an unmodifiable list while `new ArrayList()` does not.

Reply:
I don't think it matters too much. If I were to change it I would make all return an unmodifiable list. I would simply leave it like this.

Reply:
What would you like me to do here?

Reply:
I'd just use a mutuable list for consistency. Later, when properly discussed immutable objects should be made immutable.

Reply:
Ok